src/App.php

```diff
@@ -92,10 +92,10 @@ class App
 	 */
 	private $baseURL;
 
-	/**
-	 * @var string The name of the current theme
-	 */
+	/** @var string The name of the current theme */
 	private $currentTheme;
+	/** @var string The name of the current mobile theme */
+	private $currentMobileTheme;
```

Comment:
Why do we need a own field for mobile? Shouldn't it compute either the mobile or the normal theme depending on the mode? Why do we need both in one execution?

Reply:
I feel like it's the most explicit way of handling it. It doesn't assume anything about where and when themes will be set (or not) or when they'll be gotten (or not).

Reply:
Hmm.. I see that we currently handle both themes differently in the profile data array.. So you're right that we will use both fields separately..